src/libmongoc/src/mongoc/mongoc-error.h

```diff
@@ -139,6 +140,10 @@ typedef enum {
    // Azure errors
    MONGOC_ERROR_AZURE_HTTP,
    MONGOC_ERROR_AZURE_BAD_JSON,
+
+   // GCP errors
+   MONGOC_ERROR_GCP_HTTP,
+   MONGOC_ERROR_GCP_BAD_JSON
```

Comment:
Maybe these should be merged since errors are already distinguished by the error domain.